uint8 my_counter
string status